NEWS.md

```diff
@@ -1,3 +1,30 @@
+# mrgsolve 1.6.0
+
+- `D_CMT`, `ALAG_CMT`, and `simeps()` are now available to use in the `$TABLE`
+  (`$ERROR`) block (#1290). 
```

Comment:
These changes were part of the 1.5.3 release.

Reply:
Thanks; removed.